### PR TITLE
Adding a schema for the randomuser.me API service

### PR DESCRIPTION
**Pull request by rdegges**
This schema allows users to generate random user data, useful for testing, creating fake accounts, etc.

This is my first schema, so might not be best practices if ya know what I mean ;) Please let me know what you think!

### CODE REVIEW

File: randomuser/stepzen.config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "RandomUser",
+  "description": "Generates random users for testing purposes using the randomuser.me API.",
+  "categories": ["testing"]
```
Comment (carloseberhardt) [inline randomuser/stepzen.config.json]: Would you please change categories to "Dev Tools" from "testing" ? This impacts how it's displayed in the CLI ui under generators.

**Review by carloseberhardt — CHANGES_REQUESTED** [review]

File: randomuser/stepzen.config.json (hunk 1 above)
Comment (rdegges) [reply randomuser/stepzen.config.json]: Will do!

**Review by carloseberhardt — APPROVED** [review]

**Comment by carloseberhardt** [comment]
Thanks, Randall!

**Comment by rdegges** [comment]
👍

**Pull request merged**